src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { formatResults } from "./formatter";
+
+// special TS syntax: https://github.com/microsoft/TypeScript/issues/2719
+export = formatResults;
```

Comment:
Are you doing this because eslint expects a formatter to have a top level function export that is the reporter function?

Reply:
Correct, otherwise it complains that it got an object instead of a function.